backend/apps/slack/models/message.py

```
@@ -0,0 +1,84 @@
+  """Slack Message model."""
+  
+  class Meta:
+    db_table = "slack_messages"
+    verbose_name_plural = "Messages"
+    unique_together = ("conversation", "slack_message_id")
+
+  conversation = models.ForeignKey(Conversation, on_delete=models.CASCADE, related_name='messages')
+  reply_count = models.PositiveIntegerField(verbose_name="Reply Count", default=0)
+  slack_message_id = models.CharField(verbose_name="Slack Message ID", max_length=50)
+  text = models.TextField(verbose_name="Message Text", blank=True)
+  thread_timestamp = models.CharField(verbose_name="Thread Timestamp", blank=True, null=True)
```

Ruff (0.11.9)

Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)